docs/protocols.md

```diff
@@ -92,13 +92,17 @@ The CQL binary protocol is Cassandra's and Scylla's main client-facing
 protocol. Scylla supports several Scylla-only extensions to this protocol,
 described in [protocol-extensions.md](protocol-extensions.md).
 
-By default scylla listens to the CQL protocol on port 9042, which can be
-configured via the `native_transport_port` configuration option, or entirely
-disabled by setting the `start_native_transport` option to 0.
-Again these option names were chosen for backward-compatibility with Cassandra
-configuration files: They refers to CQL as the "native transport", to contrast
-with the older Thrift protocol (described below) which wasn't native to
-Cassandra.
+By default, Scylla listens to the CQL protocol on port 9042, which can be
+configured via the `native_transport_port` configuration option. Scylla also
+supports the CQL protocol via TLS/SSL encryption, which can be enabled via the
+`native_transport_port_ssl` configuration option (default port is 9142). Users
+that want to only support the encrypted CQL protocol can disable the
+unencrypted default CQL protocol by setting the `native_transport_port` to 0.
+The CQL protocol support can be disabled altogether by setting the
+`start_native_transport` option to `false`. These option names were chosen for
+backward-compatibility with Cassandra configuration files: They refers to CQL
+as the "native transport", to contrast with the older Thrift protocol
+(described below) which wasn't native to Cassandra.
```

Comment:
Also need to remove the "TODO: again mention SSL, and `native_transport_port_ssl` (default 9142)." line below.